src/main/resources/schemas/data-product-definition-schema.json

```diff
@@ -0,0 +1,274 @@
+        "condition": {
+          "type": "array",
+          "items": { "$ref": "#/definitions/policy-condition" }
+        }
+      },
+      "additionalProperties": false
+    },
+
+    "policy-condition": {
+      "anyOf": [
+        { "$ref": "#/definitions/policy-condition-match" },
+        { "$ref": "#/definitions/policy-condition-nomatch" },
+        { "$ref": "#/definitions/policy-condition-like" },
+        { "$ref": "#/definitions/policy-condition-between" }
+      ]
+    },
+
+    "policy-condition-match": {
+      "type": "object",
+      "properties": {
+        "match": {
+          "type": "array",
+          "items": { "type": "string" }
+        }
+      }
+    },
+
+    "policy-condition-nomatch": {
+      "type": "object",
+      "properties": {
+        "notequals": {
+          "type": "array",
+          "items": { "type": "string" }
+        }
+      }
+    },
+
+    "policy-condition-like": {
+      "type": "object",
+      "properties": {
+        "like": {
+          "type": "array",
+          "items": { "type": "string" }
+        }
+      }
+    },
+
+    "policy-condition-between": {
+      "type": "object",
+      "properties": {
+        "between": {
+          "type": "object",
+          "required": ["value", "start", "end"],
+          "properties": {
+            "value": { "type": "string" },
+            "start": { "type": "string" },
+            "end": { "type": "string" }
+          }
+        }
+      }
+    },
+
+    "report": {
+      "type": "object",
+      "required": ["id", "name", "version", "dataset", "render", "specification"],
+      "properties": {
+        "id": { "type": "string" },
+        "name": { "type": "string" },
+        "description": { "type": "string" },
+        "created": { "type": "string" },
+        "modified": { "type": "string" },
+        "withdrawn": { "type": "string" },
+        "version": { "type": "string" },
+        "dataset": { "type": "string" },
+        "render": { "type": "string" },
+        "classification": { "type": "string" },
+        "feature": { "type": "array", "items": { "$ref": "#/definitions/feature" } },
+        "policy": { "type": "array", "items": { "$ref": "#/definitions/policy" } },
+        "specification": { "$ref": "#/definitions/report-specification" }
+      }
+    },
+
+    "report-specification": {
+      "type": "object",
+      "required": ["template", "field"],
+      "properties": {
+        "template": { "type": "string" },
+        "field": {
+          "type": "array",
+          "items": { "$ref": "#/definitions/report-field" }
+        }
+      }
+    },
+
+    "report-field": {
+      "type": "object",
+      "required": ["name"],
+      "properties": {
+        "description": { "type": "string" },
+        "name": { "type": "string" },
+        "display": { "type": "string" },
+        "formula": { "type": "string" },
+        "visible": { "type": "boolean" },
```

Comment:
This is still a boolean.
Based on the PR title should this not be an enum?

Reply:
Resolved. The first PR was for the file.